src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -346,6 +346,7 @@ public static class SensitivityAnalysisResults
          public const string PARAMETER = "Parameter";
          public const string VALUE = "Value";
          public const string PK_PARAMETER = "PKParameter";
+         public const string PARAMETER_PATH = "ParameterPath";
```

Comment:
@Yuri05 I am also adding the ParameterPath to the export as the Parameter may not been enough